Apps/W1/APIV2/app/src/pages/APIV2FALocations.Page.al

```diff
@@ -0,0 +1,36 @@
+page 30097 "APIV2 - FA Locations"
+{
+    DelayedInsert = true;
+    APIVersion = 'v2.0';
+    EntityCaption = 'FALocation';
```

Comment:
@onbuyuka the captions are the same as are in Business Central this is wjy they are not called Fixed Asset Locations since otherwise you would have different names in both places

![image](https://github.com/microsoft/ALAppExtensions/assets/106300783/04e7cac8-354c-4154-8227-16b2e7d565a1)

![image](https://github.com/microsoft/ALAppExtensions/assets/106300783/0659d1d3-50b2-4c21-b50a-def61d09388e)

![image](https://github.com/microsoft/ALAppExtensions/assets/106300783/cb70a3fc-e286-4539-ae66-b6f93eaf7c4b)

Reply:
UI page captions are different than API page captions. They are used in integration scenarios like virtual tables in Dataverse by non-BC users. It should be understandable for anyone, BC users or non-BC users.

Reply:
Ok. Will change